src/index.html

```diff
@@ -22,6 +22,367 @@
+      >
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header-nav">
+        <ul class="header-nav__list">
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review_amount">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price_title">Price:</p>
+          <p class="card__price_sum">$2,199</p>
+        </div>
+
+        <a
+          class="card__link"
+          data-qa="card-hover"
+        >
+          <p class="buy">BUY</p>
```

Comment:
Redundant, so you need to fix it everywhere
```suggestion
          BUY
```